lib/interfaces/bdd.js

```diff
@@ -40,11 +40,14 @@ module.exports = function (suite) {
      * and/or tests.
      */
 
-    context.Feature = function (title) {
+    context.Feature = function (title, retries) {
```

Comment:
I'd make pass it as hash: `{ retries: number }`
probably we will use it in future for other configuration options